src/emulator/ContractExecutor.ts

```diff
@@ -57,6 +57,7 @@ export class ContractExecutor {
     #index = 0;
     #verbosity: Verbosity | null = null;
     #name: string | null = null;
+    #libs: Cell | null = null;
```

Comment:
btw, what problem does this solve for you?

Reply:
i need to mock a contract, which is a lib deployed on master chain.  and i get the code from master chain. but i find that i need to pass the lib along with the code, so that it can work for me